bot.py

```diff
@@ -30,8 +31,9 @@ async def on_ready():
     token = json.loads(data_file.read())['TOKEN']
 
 bot.remove_command('help')  # Overwrite normal 'help' command
-#bot.add_cog(CommandErrorHandler(bot))
+# bot.add_cog(CommandErrorHandler(bot))
```

Comment:
Just a nitpick, but to avoid an extra diff, I think this added space should be removed.